lib/internal/fs/streams.js

```diff
@@ -362,6 +373,8 @@ WriteStream.prototype._writev = function(data, cb) {
     });
   }
 
+  if (this.destroyed) return cb();
```

Comment:
I don't think you need to call the callback here. This should only happen if coming from the event listener.